src/libs/actions/PersistedRequests.js

```diff
@@ -17,7 +18,12 @@ function clear() {
  * @param {Array} requestsToPersist
  */
 function save(requestsToPersist) {
-    persistedRequests = persistedRequests.concat(requestsToPersist);
+    HttpUtils.cancelPendingReconnectAppRequests();
+    persistedRequests = _.chain(persistedRequests)
+        .concat(requestsToPersist)
+        .partition(request => request.command !== 'ReconnectApp')
```

Comment:
We're using 'ReconnectApp' string, it's easy to make a typo and it will silently not work. We could store it in `CONST.NETWORK.COMMANDS` (there might be a etter place)

Reply:
Agree, I've updated my PR like this. Thanks